app/Api/V1/Controllers/ProblemController.php

```diff
@@ -32,4 +35,29 @@ public function store(
         $service->createOne($params);
         return $this->response->created();
     }
+
+    public function index(
+        ProblemTransformer $problemTransformer,
+        ReadProblemService $service,
+        Request $request
+    ):Response
```

Comment:
Missing `space` before Response
You should try to use `ctrl + shift + l` (or `ctrl + windows + shift + l` in linux)  to autoformat your code in phpstorm

Reply:
Autoformat in phpstroms is `ctrl + alt + shift + l` :+1: